Issue 757 : Maintain state in RStudio

vkumra-broad opened the pull request:
\<your comments for this PR go here\>

Have you read [CONTRIBUTING.md](https://github.com/DataBiosphere/leonardo/blob/develop/CONTRIBUTING.md) lately? If not, do that first.

I, the developer opening this PR, do solemnly pinky swear that:

- [ ] I've documented my API changes in Swagger

In all cases:

- [ ] Get a thumbsworth of review and PO signoff if necessary
- [ ] Verify all tests go green
- [ ] Squash and merge; you can delete your branch after this
- [ ] Test this change deployed correctly and works on dev environment after deployment

vkumra-broad committed:
update the conf set in the rstudio docker

vkumra-broad committed:
nit

rtitle commented on docker/rstudio/Dockerfile:
```diff
@@ -26,4 +26,8 @@ ENV RSTUDIO_HOME /etc/rstudio
 
 ADD rserver.conf $RSTUDIO_HOME/rserver.conf
 
+RUN sed 's/alwaysSaveHistory="0"/alwaysSaveHistory="1"/g' \
```
Don't you have to pass the filename to `sed`? I would expect something like
```
RUN sed -i 's/alwaysSaveHistory="0"/alwaysSaveHistory="1"/g' <the-filename> \
  && sed -i 's/loadRData="0"/loadRData="1"/g' <the-filename> \
  && sed -i 's/saveAction="0"/saveAction="1"/g' <the-filename>
```

vkumra-broad committed:
file name, duh

vkumra-broad committed:
better sed

remove debug code

add r profile config

set rprofileOnResume

clean up

separate RUN for sed

this might do it

this should be it

MatthewBemis commented:
another scenario we should test is deleting a cluster and trying to restore that session on a new cluster

vkumra-broad commented:
Faces with @MatthewBemis :  the delete and restore on new cluster would be a good once we have gcsfuse integrated.

My manual test scenario in this case is:

1. Create cluster
2. Create a new Rstudio proj
3. Run some commands, load some data. this will populate the .Rhistory and .Rdata files. 
4. Close the session
5. Restart the cluster. 
6. Rstudio should open the previous project with data and history available from the previous session.
